packages/browser/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@formbricks/js-internal",
```

Comment:
can we name it `js-core` and also use `js-core` instead of `browser`? :-)

Reply:
Or please something like `js_core`. Because building it as a iife does not support `-` in the module name

Reply:
@jonas-hoebenreich 👍 What was the reason again why you prefer the `iife` format?

Reply:
We initially had issues with umd leaking properties to the window object (which crashed our react shop). I just checked and it seems like this is no longer the case except the surveys package adding formbricks-surveys, formbricksSurveys & DOMPurify (below my code for reference).
We know that iife works for us, so before a switch to umd we would have to evaluate it again.

```
var initialWindowProperties = Object.keys(window);
var j = document.createElement('script');
j.async = true;
j.defer = true;
j.src = 'https://unpkg.com/@formbricks/surveys@1.99.0/dist/index.umd.js';
document.body.appendChild(j);
function logNewWindowProperties() {
    console.log("New Window Properties:");
    var currentWindowProperties = Object.keys(window);
    currentWindowProperties.forEach(function(prop) {
        if (!initialWindowProperties.includes(prop)) {
            console.log(prop);
        }
    });
}
j.onload = logNewWindowProperties;
```

Reply:
hey @jonas-hoebenreich, I just tested the package with iife with the same name and it worked locally! Anything else that I can test with?